Dockerfile

```diff
@@ -76,6 +77,7 @@ RUN \
         .. && \
     make -j$(nproc) chain_test test_fixed_string plugin_test && \
     ./tests/chain_test && \
+    ./tests/plugin_test -t json_rpc && \
     ./programs/util/test_fixed_string && \
```

Comment:
Why aren't we running all plugin tests?

Reply:
Some are broken. Because the plugin tests were never ran, they weren't updated alongside other code. I'll file a ticket to fix them, but I don't believe it should be a blocker for this PR.

Reply:
For this, I was mistaken. All plugin tests pass. Will change in the Dockerfile